tests/tests_cleanup.yml

```diff
@@ -0,0 +1,61 @@
+- hosts: all
+  tasks:
+    - name: Set version specific variables
+      include_vars: "{{ lookup('first_found', ffparams) }}"
+      vars:
+        ffparams:
+          files:
+            - "{{ ansible_facts['distribution'] }}_\
+              {{ ansible_facts['distribution_version'] }}.yml"
+            - "{{ ansible_facts['distribution'] }}_\
+              {{ ansible_facts['distribution_major_version'] }}.yml"
+            - "{{ ansible_facts['distribution'] }}.yml"
+            - "{{ ansible_facts['os_family'] }}.yml"
+            - "default.yml"
+          paths:
+            - "{{ playbook_dir }}/roles/linux-system-roles.vpn/vars"
+
+    - name: check if ipsec service exists
+      stat: path=/etc/init.d/ipsec
+      register: ipsec_serv_status
+
+    - name: stop ipsec service
+      service:
+        name: ipsec
+        state: stopped
+        enabled: no
+      when: ipsec_serv_status.stat.exists
```

Comment:
do you need the `when` check here?  If so, you should use the `service_facts` module to get the service facts to see if the service is enabled and started https://docs.ansible.com/ansible/latest/collections/ansible/builtin/service_facts_module.html#ansible-collections-ansible-builtin-service-facts-module